shell/platform/windows/flutter_windows_engine.h

```diff
@@ -192,6 +195,9 @@ class FlutterWindowsEngine {
   FlutterDesktopOnPluginRegistrarDestroyed
       plugin_registrar_destruction_callback_ = nullptr;
 
+  // Collection of engine switches.
+  std::vector<std::string> switches_;
```

Comment:
Why are you adding storage of switches in another place, instead of keeping them with the dart project where the conceptually live now?